client/account-recovery/lost-password/style.scss

```diff
@@ -0,0 +1,24 @@
+.lost-password__title {
+	font-size: 1.5em;
+	font-weight: bold;
+}
+
+.lost-password__user-login-input {
+	//Use a more specific selector so we don't have to use !important to overide any css
+	&.form-text-input {
```

Comment:
My initial intention was to be verbose since it all gets compiled into the same CSS but I changed it in https://github.com/Automattic/wp-calypso/pull/8955/commits/aa0fb82810a4cea03761e10cfcf87779ad6b8add